src/path.js

```diff
@@ -16,6 +16,283 @@ function Path() {
     this.strokeWidth = 1;
 }
 
+function roundDecimal(float, places) {
+    return +(Math.round(float + 'e+' + places) + 'e-' + places);
+}
+
+function optimizeCommands(commands) {
+    // separate subpaths
+    let subpaths = [[]];
+    for (let i = 0; i < commands.length; i += 1) {
+        const subpath = subpaths[subpaths.length - 1];
+        const cmd = commands[i];
+        const firstCommand = subpath[0];
+        const secondCommand = subpath[1];
+        const previousCommand = subpath[subpath.length - 1];
+        subpath.push(cmd);
+        if (cmd.type === 'Z') {
+            // When closing at the same position as the path started,
+            // remove unnecessary line command
+            if (
+                firstCommand.type === 'M' &&
+                secondCommand.type === 'L' &&
+                previousCommand.type === 'L' &&
+                previousCommand.x === firstCommand.x &&
+                previousCommand.y === firstCommand.y
+            ) {
+                subpath.shift();
+                subpath[0].type = 'M';
+            }
+
+            if (i + 1 < commands.length) {
+                subpaths.push([]);
+            }
+        } else if (cmd.type === 'L') {
+            // remove lines that lead to the same position as the previous command
+            if (previousCommand.x === cmd.x && previousCommand.y === cmd.y) {
+                subpath.pop();
+            }
+        }
+    }
+    commands = [].concat.apply([], subpaths); // flatten again
+    return commands;
+}
+
+/**
+ * Returns options merged with the default options for parsing SVG data
+ * @param {object} options (optional)
+ */
+function defaultSVGParsingOptions(options) {
```

Comment:
wouldn't it be better to call this createSVGParsingOptions, because if we're overriding the defaults it's not default.

Reply:
True, I'll change that!